product_packaging_dimension/models/product_packaging.py

```diff
@@ -6,7 +6,7 @@
 class ProductPackaging(models.Model):
     _inherit = "product.packaging"
 
-    max_weight = fields.Float()
+    weight = fields.Float()
```

Comment:
:warning: We might have to drop commits 4d877ab and 6bfbea7 because `max_weight` is defined in `delivery`: https://github.com/odoo/odoo/blob/12.0/addons/delivery/models/product_packaging.py#L13